### PR TITLE
fix to enable using UUIDs/non-default PKs

### PR DESCRIPTION
**Pull request by PetrDlouhy**
I tried to use UUIDs as PKs as mentioned in #179, and the only problem preventing that was these two small misuses of ID instead of PK.

### CODE REVIEW

**Comment by PetrDlouhy** [comment]
@danirus I have rebased this PR to current master nad added a test case for `commentbox_props_respons()` (which was untested until now).

This is very simple change which fixes #179 (with exception of Proxy models) and should be harmless otherwise.

**Comment by PetrDlouhy** [comment]
The changes in code for this are overriden by #374, although the tests should be still valid and work with the new code there.

File: django_comments_xtd/api/frontend.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class CommentBoxDriver(object):
     def get_queryset(cls, ctype, obj, request):
         return XtdComment.objects.filter(
             content_type=ctype,
-            object_pk=obj.pk,
+            object_pk=obj._get_pk_val(),
```
Comment (danirus) [inline django_comments_xtd/api/frontend.py:28]: I'm not sure why this change is needed. The `pk` attribute is defined [here](https://github.com/django/django/blob/3e3e8141662c70d74d3aaef9d4b053c9636ea0d7/django/db/models/base.py#L680) as a property that resolves to `_get_pk_val` for the getter and to `_set_pk_val` for the setter. So using `obj.pk` calls `obj._get_pk_val()`. What am I missing?

Comment (PetrDlouhy) [reply django_comments_xtd/api/frontend.py:28]: I am not sure. Probably it was needed when I was exchanging the function in my hacky implementation of Proxy models.
But also might not be needed even then and I changed it because I taught it was needed.

I will change it to pure `pk`s and will see if everything works.
